python_modules/dagster/dagster/_core/host_representation/external_data.py

```diff
@@ -1069,6 +1076,33 @@ def __new__(
         )
 
 
+@whitelist_for_serdes
+class ExternalAssetCheck(
+    NamedTuple(
+        "_ExternalAssetCheck",
+        [
+            ("name", str),
+            ("asset_key", AssetKey),
+            ("description", Optional[str]),
+        ],
```

Comment:
I think this will need to contain something like `op_name` that will let us differentiate between
- checks that run in isolated ops
- checks that run in the same op as the materialization (and thus can't be run separately)
- multi checks